library/common/2106.0.0/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
+name: common
+description: A library chart for iX Official Catalog
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: library
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 2106.0.0
```

Comment:
The naming scheme for library chart follows `YYMM` pattern, so let's please adjust it to `2112` in this case

Reply:
Also let's please update all other apps to use the newer library version. We have a script `update_dependencies` which can assist with this workflow.